editor.planx.uk/src/@planx/components/Feedback/components/placeholders.tsx

```diff
@@ -0,0 +1,15 @@
+export const freeformQuestionPlaceholder =
+  "Please tell us more about your experience.";
+
+export const ratingQuestionPlaceholder =
+  "How would you rate your experience with this service?";
+
+export const descriptionPlaceholder = `This service is a work in progress, any feedback you share about your experience will help us to improve it.
+<br>
+<br>
+Don't share any personal or financial information in your feedback. If you do we will act according to our <a href="">privacy policy</a>.`;
+
+export const disclaimerPlaceholder =
+  "The information collected here isn't monitored by planning officers. Don't use it to give extra information about your project or submission. If you do, it cannot be used to assess your project.";
+
+export const titlePlaceholder = "Tell us what you think";
```

Comment:
nit: I'm slighty hesitant about introducing "placeholders" file naming into our structure as this is going to mean something new soon in the world of "templates" ! 

Would the `defaultContent` structure currently used in `@planx/components/DrawBoundary/model` work here? Has small benefit of also being type-able based on component prop types :slightly_smiling_face:

Reply:
Thanks for all these suggestions! They're in the next PR :)